sensing/radar_scan_to_pointcloud2/src/radar_scan_to_pointcloud2_node/radar_scan_to_pointcloud2_node.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "radar_scan_to_pointcloud2/radar_scan_to_pointcloud2_node.hpp"
+
+#include <pcl/pcl_base.h>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+#include <pcl_conversions/pcl_conversions.h>
+
+using namespace std::literals;
+using std::chrono::duration;
+using std::chrono::duration_cast;
+using std::chrono::nanoseconds;
+using std::placeholders::_1;
+
+namespace
+{
+template <class T>
+bool update_param(
+  const std::vector<rclcpp::Parameter> & params, const std::string & name, T & value)
+{
+  const auto itr = std::find_if(
+    params.cbegin(), params.cend(),
+    [&name](const rclcpp::Parameter & p) { return p.get_name() == name; });
+  // Not found
+  if (itr == params.cend()) {
+    return false;
+  }
+  value = itr->template get_value<T>();
+  return true;
+}
+
+pcl::PointXYZI getPointXYZI(const radar_msgs::msg::RadarReturn & radar, float intensity)
+{
+  const float x = radar.range * std::cos(radar.azimuth) * std::cos(radar.elevation);
+  const float y = radar.range * std::sin(radar.azimuth) * std::cos(radar.elevation);
+  const float z = radar.range * std::sin(radar.elevation);
```

Comment:
It maybe better to use a temporary variable because cos, sin have heavy cost.

Reply:
fixed at [make temporary variable](https://github.com/autowarefoundation/autoware.universe/pull/2336/commits/073377d85bc1079a7b6c22ae6600d687d6434996)